schemas/test/en/test_supplementary_data_all_variations.json

```diff
@@ -0,0 +1,934 @@
+{
```

Comment:
We should enable view submitted response and ensure that it renders fine.

Can enable using 

```json
"post_submission": {
    "view_response": true
},
```